src/test/java/com/hocs/test/glue/mts/CaseDetailsStepDefs.java

```diff
@@ -0,0 +1,31 @@
+package com.hocs.test.glue.mts;
+
+import com.hocs.test.pages.BasePage;
+import com.hocs.test.pages.mts.CaseDetails;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+
+public class CaseDetailsStepDefs extends BasePage {
+
+    CaseDetails caseDetails;
+
+    @And("I complete the Case Details stage and close the MTS Case")
+    public void iCompleteTheCaseDetailsStageOfTheMTSCase() {
```

Comment:
I think this should be 'I complete the Data Input stage and close the MTS Case'. Also I think the file names should be after the stage and not the page title